eks-worker-al2.json

```diff
@@ -29,7 +29,9 @@
     "ssh_username": "ec2-user",
     "temporary_security_group_source_cidrs": "",
     "associate_public_ip_address": "",
-    "subnet_id": ""
+    "subnet_id": "",
+    "remote_folder": "",
```

Comment:
Should you set this to `/tmp` as a default, so it gets propagated to the `install-worker.sh` and elsewhere?

Reply:
Hi @micahhausler ,
Thanks for the review. 

As per the packer doc, the remote_folder is just where the "install-worker.sh" script gets uploaded to and then executed from:
https://www.packer.io/docs/provisioners/shell.html#remote_folder

In this case, /tmp/worker is just used as a staging folder for the other config files (iptables etc) and nothing is executed out of it.

Also, if nothing is specified in the variables for remote_folder, it just defaults to /tmp so its ok to leave it blank.

I have tested my changes on a couple of image builds and it works fine.

Cheers.

Reply:
You're right that `/tmp` is the packer default, but the `install-worker.sh` script won't know where the default is. It will be easiest to follow if the default here is `/tmp`, and then you can plumb that value down to the bash script

Reply:
`remote_folder` in this scenario is where packer will upload the script that it generates. So for example when this runs, packer creates a file that will have the contents of what's in the inline block on the host:
`/tmp/random-name.sh`
```
#!/bin/bash
mkdir -p /tmp/worker
```
So in this case the `install-worker.sh` script doesn't need to know where packer is putting it's executables as all the references in `install-worker.sh` are absolute and aren't affected by where the script is bring ran.

Reply:
Any updates on this one? Our team could really use this change as we're using a CIS hardened Amazon Linux AMI which doesn't allow exec from `/tmp`.

Reply:
Ah got it